data.sql

```diff
@@ -3,3 +3,12 @@
 INSERT INTO animals (name) VALUES ('Luna');
 INSERT INTO animals (name) VALUES ('Daisy');
 INSERT INTO animals (name) VALUES ('Charlie');
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES ('Agumon', '2020-02-03', 0, true, 10.23);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES ('Gabumon', '2018-12-15', 2, true, 8);
```

Comment:
Kindly note that `Gabumon` was born on Nov 15th, 2018 as shown in the project requirements below. Please update accordingly. :+1: 

![Screenshot from 2023-08-21 12-06-59](https://github.com/RileyManda/vet-clinic/assets/65233860/a22c119a-45e6-457b-bb2a-ab1f890f1b03)